### ISSUE TITLE
I can Update and Delete activities from a Skill (JS)

### ISSUE THREAD

**Issue by clearab** [microsoft/botbuilder-js#1995]
See [parent](https://github.com/microsoft/botframework-sdk/issues/5788)

**Comment by stevengum** [comment]
Waiting on https://github.com/microsoft/botbuilder-dotnet/pull/4786 to land

**Comment by gabog** [comment]
FYI @stevengum, dotnet has been merged into master
